js/multiUserEditing.js

```diff
@@ -90,7 +90,7 @@ jQuery.entwine("multiUserEditing", function($) {
 				usersBR += "</ul>";
 
 				var dotHTML = '<div class="user-label multi-user-icon" title="' + users +
-					'"><img height="5" width="5" ' + 'src="multiuser-editing/images/' + dot + '"></div>';
+					'"><img height="5" width="5" ' + 'src="resources/silverstripe/multiuser-editing/images/' + dot + '"></div>';
```

Comment:
These hard-coded paths could be an issue - but I think we should leave them for now as I'm not sure the best way to resolve these paths in JS